iceoryx_dds/test/moduletests/test_toml_gateway_config_parser.cpp

```diff
@@ -157,29 +157,6 @@ TEST_F(TomlGatewayConfigParserTest, FailsValidationIfNoEventNameInServiceDescrip
     }
 }
 
-TEST_F(TomlGatewayConfigParserTest, FailsValidationIfNoDataSize)
-{
-    // ===== Setup
-    // Prepare configuration to test with
-    auto toml = cpptoml::make_table();
-    auto serviceArray = cpptoml::make_table_array();
-
-    auto serviceEntryNoSize = cpptoml::make_table();
-    serviceEntryNoSize->insert("service", "service");
-    serviceEntryNoSize->insert("instance", "instance");
-    serviceEntryNoSize->insert("event", "event");
-    serviceArray->push_back(serviceEntryNoSize);
-    toml->insert("services", serviceArray);
-
-    // ===== Test
-    auto result = iox::dds::StubbedTomlGatewayConfigParser::validate(*toml);
-    EXPECT_EQ(true, result.has_error());
-    if (result.has_error())
-    {
-        EXPECT_EQ(iox::dds::TomlGatewayConfigParseError::INCOMPLETE_SERVICE_DESCRIPTION, result.get_error());
```

Comment:
Why is this test obsolete?

Reply:
Data size no longer needs to be provided as part of the gateway config. The size is automatically detected by checking the payload size of DDS samples.